webapi/src/main/scala/org/knora/webapi/core/LayersLive.scala

```diff
@@ -105,7 +106,8 @@ object LayersLive {
       ConstructTransformer.layer,
       DspIngestClientLive.layer,
       GravsearchTypeInspectionRunner.layer,
-      GroupsResponderADMLive.layer,
+      GroupsResponderADM.layer,
+      GroupsRestService.layer,
```

Comment:
```suggestion
```

**issue**: Creating the same service twice

This service is already present in the [AdminApiModule](https://github.com/dasch-swiss/dsp-api/blob/main/webapi/src/main/scala/org/knora/webapi/slice/admin/api/AdminApiModule.scala#L54)